src/debian/manifest.json

```diff
@@ -175,6 +175,31 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/debian/12/helix/amd64",
+              "os": "linux",
+              "osVersion": "bookworm",
+              "tags": {
+                "debian-12-helix-amd64-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
+                "debian-12-helix-amd64$(FloatingTagSuffix)": {}
+              }
+            }
+          ]
+        },
+        {
```

Comment:
This should be inside the above platforms array

Reply:
It wasn't obvious to me how these arrays are intended to work. Can you help with understand that?

Reply:
I think that @mthalman would do a better job, especially as it seems I still got my suggestion wrong. I think each image definition gets its own outer platform []. but tbh, not sure.

Reply:
Got it. I pushed a change for that.